app/proguard-rules.pro

```diff
@@ -73,3 +73,10 @@
 -dontwarn org.conscrypt.**
 # A resource is loaded with a relative path so the package of this class must be preserved.
 -keepnames class okhttp3.internal.publicsuffix.PublicSuffixDatabase
+
+####################################################################################################
+# Glean
+####################################################################################################
+# Keep code generated from Glean Metrics
+-keep class org.mozilla.focus.GleanMetrics.** {  *; }
+-dontwarn mozilla.telemetry.glean.testing.**
```

Comment:
The `GleanTest*` classes are part of the Glean Testing APIs. They are packed in the Glean SDK library for different reasons (mostly because we can't easily split test/non-APIs for Glean). Unless you are using the glean testing APIs in your tests, any proguard warning about these classes can be safely ignored.

We should probably add this line to our own proguard.

Reply:
Filed [this on our end](https://bugzilla.mozilla.org/show_bug.cgi?id=1655116). Good catch!